### DISCUSSION TITLE
soup update error because /boot is full

### DISCUSSION THREAD

**Discussion by ponte19** [Security-Onion-Solutions/securityonion#12457] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

256 GB

### Storage for /

2TB

### Storage for /nsm

1,8TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I wanted to run `soup` to update the security onion server but it failed because of not enough disk space in `/boot`.

In `/boot` there are several old `initramfs-X.XX.X-` and `vmlinuz-X.XX.X-` files.        
What is the recommended way to remove these old kernels to free up space for `soup` ?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
In 2.4.40 we switched the ISO to set the /boot partition to 1G since others were running into this issue, but we were unable to replicate it ourselves.

Others have manually removed old kernels you may have to do the same keeping the current and 1 version older.https://github.com/Security-Onion-Solutions/securityonion/discussions/11859
